src/main/resources/assets/minestuck/models/item/cassette_retro_battle.json

```diff
@@ -0,0 +1,61 @@
+{
+	"credit": "Made with Blockbench",
```

Comment:
Since all cassettes use the same model, I'd suggest seperating the model part into its own abstract model that the others extend and define textures for. Also unsure about the resolution of the cassette items.

Reply:
it's a cool idea and I will do it, but need to find someone to redo the model before doing so.